project/Dependencies.scala

```diff
@@ -7,7 +7,7 @@ object Dependencies {
 
   val googleV = "1.22.0"
   val olderGoogleV = "1.20.0"   // TODO why do we have two google versions?  GAWB-2149
-  val workbenchGoogle2V = "0.6-4df42ca"
+  val workbenchGoogle2V = "0.10-bd284e7"
```

Comment:
all the exclude rules below are a result of updating this library; its transitive dependencies caused some conflicts.